docs/deployment/visual-studio-properties.md

```diff
@@ -0,0 +1,666 @@
+
+## Output
+
+Settings related to generating the output file.
+
+- **Base output path**
+
+  Specifies the base location for the project's output during build. Subfolders will be appended to this path to differentiate project configuration.
+
+  Defaults to `.\bin\`.
+
+- **Base intermediate output path**
+
+  Specifies the base location for the project's intermediate output during build. Subfolders will be appended to the path to differentiate project configuration.
+
+  Defaults to `.\obj\`.
+
+- **Reference assembly**
+
+  When enabled, produces a reference assembly containing the public API of the project. This is disabled by default.
+
+- **Documentation file**
+
+  When enabled, generates a file containing API documentation. This is disabled by default.
+
+### Events
+
+In this section you can add commands that run during the build.
+
+- **Pre-build event**
+
+  Specifies commands that run before the build starts. Does not run if the project is up-to-date. A non-zero exit code will fail the build before it runs.
+
+- **Post-build event**
+
+  Specifies commands that run before the build starts. Does not run if the project is up-to-date. A non-zero exit code will fail the build before it runs.
+
+- **When to run the post-build event**
+
+  Specifies under which condition the post-build even will be run.
+
+### Strong naming
+
+Settings related to signing the assembly.
+
+- **Sign the assembly**
+
+  When enabled, signs the output assembly to give it a strong name.
+
+### Advanced
+
+Additional settings related to the build.
+
+- **Language version**
+
+  The version of the language available to the code in the project. Defaults to `10.0`.
+
+- **Check for arithmetic overflow**
+
+  Throw exceptions when integer arithmetic produces out of range values. This setting is available for each platform. The default is disabled for each platform.
+
+- **Deterministic**
+
+  Produce identical compilation output for identical inputs. This setting is available for each platform. The default is enabled for each platform.
+
+- **Internal compiler error reporting**
+
+  Send internal compiler error reports to Microsoft. Defaults to `Prompt before sending`.
+
+- **File alignment**
+
+  Specifies, in bytes, where to align the sections of the output file. This setting is available for each platform. The default is `512` for each platform.
+
+## Package
+
+The **Package** section describes settings related to generating a NuGet package.
+
+## General
+
+Settings related the NuGet package.
+
+- **Generate NuGet package on build**
+
+  When enabled, produces a NuGet package file during build operations. This is disabled by default.
+
+- **Package ID**
+
+  The case-insensitive package identifier, which must be unique across nuget.org or whatever gallery the package resides in. IDs may not contain spaces or characters that aren't valid for a URL, and generally follow .NET namespace rules.
```

Comment:
"or whatever gallery" > "or the gallery"?

Reply:
```suggestion
  The case-insensitive package identifier, which must be unique across the NuGet package gallery, such as nuget.org. IDs may not contain spaces or characters that aren't valid for a URL, and generally follow .NET namespace rules.
```